### DISCUSSION TITLE
Preventing takeover in the future

### DISCUSSION THREAD

**Discussion by terrarier2111** [PrismLauncher/PrismLauncher#17] [Q&A]
How are you guys planning to prevent a takeover situation in the future?

**Comment by benjamin051000** [comment]
🦗🦗 

I guess this is a GH organization which is new... I am unsure of how this would prevent issues though. I don't think there is anything particularly groundbreaking about org permissions compared to a normal repo owned by a single person.

**Comment by benjamin051000** [comment]
I asked them on mastodon: https://mastodon.social/@benjamincodes/112283067723412471

hopefully we will get a response

**Comment by roke-julian-lockhart** [comment]
> [`#discussion-4482586`](https://github.com/orgs/PrismLauncher/discussions/17#discussion-4482586)

@terrarier2111, [`discussions/529#discussion-4605188`](https://github.com/orgs/PrismLauncher/discussions/529#discussion-4605188) contains some relevant discourse.

I believe that the sole decent way to prevent this is something akin to what Proton Mail has just done. Specifically, they've made their company a subsidiary of a Swiss charity, [^1] which is legally bound to adhere to its stated mission goals. All that this community would need to do is found a charity of the same nature with some basic rules. However, that's a fair bit to ask of some software developers.

[^1]: [`reddit.com/r/ProtonMail/comments/1dhwsx3/comment/l90ad9j`](https://www.reddit.com/r/ProtonMail/comments/1dhwsx3/comment/l90ad9j/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)

Aforestated alternatives, like a shared account, aren't feasible. [^2]

[^2]: [`discussions/529#discussioncomment-9834075`](https://github.com/orgs/PrismLauncher/discussions/529#discussioncomment-9834075)

**Comment by glkdrlgkrlzflnjkgh** [comment]
huh.
